src/interp.rs

```diff
@@ -59,7 +46,7 @@ impl Interp {
     }
 }
 
-mod generic {
+pub mod generic {
```

Comment:
This is not used at all anymore. The generic impls also use the specialized 2/4 factor impls :)

Reply:
Ah I see, you remove it in the next commit. Feels like those two commits should be squashed then :)